lua/lspconfig/server_configurations/azure_pipelines_ls.lua

```diff
@@ -1,8 +1,15 @@
 local util = require 'lspconfig.util'
 
+local bin_name = 'azure-pipelines-language-server'
```

Comment:
I don't think the npm package provide this executable. It is rather a mapping from your registry https://github.com/mason-org/mason-registry/blob/main/packages/azure-pipelines-language-server/package.yaml

Reply:
https://github.com/microsoft/azure-pipelines-language-server/pull/147, but it seems like they haven't made a release yet

Reply:
oh, it's cool then